Chapter08/recipe-06/cxx-example/src/evolution/CMakeLists.txt

```diff
@@ -0,0 +1,26 @@
+target_sources(
+  automaton
+  PRIVATE
+    ${CMAKE_CURRENT_LIST_DIR}/evolution.cpp
+    ${CMAKE_CURRENT_LIST_DIR}/evolution.hpp
```

Comment:
Same comment as before, consider whether the header should be part of the `PUBLIC` sources.